volume/validate.go

```diff
@@ -135,18 +135,18 @@ func (e *errMountConfig) Error() string {
 }
 
 func errExtraField(name string) error {
-	return fmt.Errorf("field %s must not be specified", name)
+	return errors.Errorf("field %s must not be specified", name)
 }
 func errMissingField(name string) error {
-	return fmt.Errorf("field %s must not be empty", name)
+	return errors.Errorf("field %s must not be empty", name)
```

Comment:
I'd have to look at it further, but don't want to keep modifying this.
I only changed this to `errors.Errorf` so that we get the stack with the error.